pkg/api/validation/validation.go

```diff
@@ -419,6 +423,17 @@ func validateGlusterfs(glusterfs *api.GlusterfsVolumeSource) errs.ValidationErro
 	return allErrs
 }
 
+func validateCinderVolumeSource(PD *api.CinderVolumeSource) errs.ValidationErrorList {
```

Comment:
rename the PD variable to something meaningful here.  it's not a PD (and it should have been 'pd' anyway)